src/frontend/src/pages/TeamsPage/ProjectCardsView.tsx

```diff
@@ -0,0 +1,31 @@
+import { Card, CardContent, Typography, Link } from '@mui/material';
+import { ProjectPreview } from 'shared';
+import { routes } from '../../utils/routes';
+import { fullNamePipe, wbsPipe } from '../../utils/pipes';
+import { useSingleProject } from '../../hooks/projects.hooks';
+import { Link as RouterLink } from 'react-router-dom';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+
+interface ActiveProjectCardViewProps {
+  project: ProjectPreview;
+}
+const ActiveProjectCardView: React.FC<ActiveProjectCardViewProps> = ({ project }) => {
+  const { isLoading, isError, error, data } = useSingleProject(project.wbsNum);
+  if (isError) return <ErrorPage message={error?.message} />;
+  if (isLoading || !data) return <LoadingIndicator />;
+
+  return (
+    <Card sx={{ width: 345 }}>
```

Comment:
for the visuals of this, copy how reid has updated the team cards becuase those look very good